subsys/disk/Kconfig

```diff
@@ -110,6 +110,21 @@ config DISK_ACCESS_SPI_SDHC
 	help
 	  File system on a SDHC card accessed over SPI.
 
+config DISK_ACCESS_NXP_USDHC1
```

Comment:
Make this:
```
config DISK_ACCESS_USDHC
	bool "NXP i.MXRT USDHC driver"
	depends on HAS_MCUX_USDHC
	help
	  Enable support for the i.MXRT USDHC driver

config DISK_ACCESS_USDHC1
	bool "NXP i.MXRT USDHC instance 1"
	depends on HAS_MCUX_USDHC
	help
	  Enable support for the i.MXRT USDHC driver instance 1

config DISK_ACCESS_USDHC2
	bool "NXP i.MXRT USDHC instance 2"
	depends on HAS_MCUX_USDHC
	help
	  Enable support for the i.MXRT USDHC driver instance 2
```

This will allow you to make the `subsys/disk/CMakeLists.txt` look like this:
```
zephyr_sources_ifdef(CONFIG_DISK_ACCESS_USDHC disk_access_usdhc.c)
```